packer.json

```diff
@@ -24,7 +24,8 @@
       },
       "run_tags": {
         "application": "{{user `app_name` }}",
-        "Name": "packer-ami-build"
+        "Name": "packer-ami-build",
+        "Env": "packer-ami-build"
```

Comment:
I believe adding this is necessary, otherwise `get_env()` will raise an exception. Not sure if we need both this and `Name` value though?